About/About/who-are-we.md

```diff
@@ -1,6 +1,6 @@
 ## Who are we?
 
-CHAOSS is a Linux Foundation project focused on creating analytics and metrics to help define community health. Work in the CHAOSS Project community is largely organized around software and metrics. Additionally, users groups provide ways to consider how software and practices can support the deployment of CHAOSS metrics. 
+CHAOSS is an open source project at the Linux Foundation focused on creating analytics and metrics to help define community health. Work in the CHAOSS Project community is largely organized around software and metrics. Additionally, users groups provide ways to consider how software and practices can support the deployment of CHAOSS metrics. 
```

Comment:
```suggestion
CHAOSS is an open source project at the Linux Foundation focused on creating analytics and metrics to help define community health of open source projects. Work in the CHAOSS Project community is largely organized around software and metrics. Additionally, users groups provide ways to consider how software and practices can support the deployment of CHAOSS metrics. 
```

Not the best wording, but differentiating to community health as a medical concern.

Reply:
I think this reads well :) Gets right to the point!